src/analyses/ai.h

```diff
@@ -274,6 +274,14 @@ class ait:public ai_baset
     ai_baset::clear();
   }
 
+  // this one just finds states
+  virtual const bool state_exist(locationt l) const
```

Comment:
This piece of code was already removed in  https://github.com/lucasccordeiro/cbmc/tree/dependence-graph-fix